x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -309,6 +330,18 @@ final int getKeySize(OptionSet options) {
             }
         }
 
+        final List<String> getCaKeyUsage(OptionSet options) {
+            if (options.has(caKeyUsageSpec)) {
+                String rawCaKeyUsage = caKeyUsageSpec.value(options);
```

Comment:
Should this be `values` so that users can pass `--ca-keyusage=keyCertSign --ca-keyusage=keyAgreement` (as is the case with `--dns` and `--ip`)

Reply:
Much nicer! Thanks for the suggestion. I've pushed https://github.com/elastic/elasticsearch/pull/126376/commits/5fdda888ae611cfd57364106e3599c655a4916dd